packages/expo-sharing/ios/SharingModule.swift

```diff
@@ -36,6 +36,18 @@ public final class SharingModule: Module {
         throw MissingCurrentViewControllerException()
       }
 
+      if UIDevice.current.userInterfaceIdiom == .pad {
+        let viewFrame = currentViewcontroller.view.frame
+        activityController.popoverPresentationController?.sourceRect = CGRect(
```

Comment:
Is setting the `sourceRect` necessary? [Apple docs](https://developer.apple.com/documentation/uikit/uipopoverpresentationcontroller/1622324-sourcerect) states:
> In iOS 13.2 and later, the default value is [CGRectNull](https://developer.apple.com/documentation/coregraphics/cgrectnull), which instructs the system to use the current frame of sourceView. The controller observes changes to this frame and updates the popover accordingly

So I'm guessing it should work fine enough 🤔

Reply:
This is the error without this
`*** Terminating app due to uncaught exception ‘NSGenericException’, reason: ‘Your application has presented a UIActivityViewController (<UIActivityViewController: 0x7f829b854600>). In its current trait environment, the modalPresentationStyle of a UIActivityViewController with this style is UIModalPresentationPopover. You must provide location information for this popover through the view controller’s popoverPresentationController. You must provide either a sourceView and sourceRect or a barButtonItem. If this information is not known when you present the view controller, you may provide it in the UIPopoverPresentationControllerDelegate method -prepareForPopoverPresentation.’`

sourceRect or sourceView needs to be specified for the popover to be visible. sourceView isn't enough in our case. The view still does not appear so I used the same approach as [here](https://github.com/expo/expo/blob/9ddb76441ee079486205de8a311800ff6618773e/packages/expo-document-picker/ios/DocumentPickerModule.swift#L35)